Makefile

```diff
@@ -156,6 +157,7 @@ uninstall-hub: manifests kustomize ## Uninstall hub CRDs from the K8s cluster sp
 	$(KUSTOMIZE) build --load_restrictor none config/hub/crd | kubectl delete -f -
 
 deploy-hub: manifests kustomize ## Deploy hub controller to the K8s cluster specified in ~/.kube/config.
+	cd config/hub/default && $(KUSTOMIZE) edit set image kube-rbac-proxy=$(RBAC_PROXY_IMG)
```

Comment:
Setting it before manager directory kustomization collects the resource works. Thanks!